godot_project/editor/rendering/structure_preview/structure_preview.gd

```diff
@@ -92,14 +90,3 @@ func set_transparency(in_transparency: float) -> void:
 func _on_representation_settings_changed(in_representation_settings: RepresentationSettings) -> void:
 	if is_instance_valid(_atomic_structure_renderer):
 		_atomic_structure_renderer.change_representation(in_representation_settings.get_rendering_representation())
-
-func _on_workspace_context_started_creating_object(workspace_context: WorkspaceContext) -> void:
-	var peek_new_object: Callable = func(in_context: StructureContext) -> bool:
-		set_structure(in_context.nano_structure)
-		return true
-	workspace_context.peek_context_of_object_being_created(peek_new_object)
-
-
-func _on_workspace_context_aborted_creating_object() -> void:
-	if is_instance_valid(_atomic_structure_renderer):
-		_atomic_structure_renderer.queue_free()
```

Comment:
I don't remember why this was needed initially, but it doesn't seem to be used anymore and it's causing the second bug mentioned in this PR.